File: user-api/ts-sdk/src/sdk/models/shared/useractivationpayload.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class UserActivationPayload extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=display_name" })
  displayName?: string;

  @SpeakeasyMetadata({ data: "json, name=password" })
  password?: string;
}

File: user-api/python-sdk/src/epilot/models/shared/useractivationpayload.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from epilot import utils


@dataclass_json
@dataclasses.dataclass
class UserActivationPayload:
    display_name: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('display_name') }})
    password: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('password') }})
    
